packs/equipment/fairy-bullet.json

```diff
@@ -11,7 +11,7 @@
         "containerId": null,
         "damage": null,
         "description": {
-            "value": "<p><strong>Ammunition</strong> round</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> command, envision</p>\n<p>These bullets are seaweed-green, refract light like an emerald, and are covered in a thin film of gray-green powder. When fired, a fairy bullet creates an effect that functions as the @UUID[Compendium.pf2e.spells-srd.Item.Revealing Light] spell in a line between you and your target. This line can be no more than 60 feet long, so if the target is more than 60 feet away from you, the line simply ends after reaching its maximum distance. Since the <em>fairy bullet</em> is fired before the <em>glitterdust</em> can reveal the target, the effects don't affect the flat check for the attack roll with the <em>fairy bullet</em>.</p>"
+            "value": "<p><strong>Ammunition</strong> round</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> command, envision</p>\n<hr />\n<p>These bullets glimmer with emerald green light that dances across the surface of the bullet like a mischievous sprite. On a successful Strike, a fairy bullet casts @UUID[Compendium.pf2e.spells-srd.Item.Revealing Light] (DC 23) extending outward from a corner of the target's space. You choose which corner of the target's space you want the burst to extend out from at the time you declare the associated Strike. Since the fairy bullet is fired before <em>revealing light</em> can reveal the target, the effects don't affect the flat check for the attack roll with the fairy bullet if the target is @UUID[Compendium.pf2e.conditionitems.Item.Hidden] from you.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Ammunition</strong> round</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> command, envision</p>\n<hr />\n<p>These bullets glimmer with emerald green light that dances across the surface of the bullet like a mischievous sprite. On a successful Strike, a <em>fairy bullet</em> casts @UUID[Compendium.pf2e.spells-srd.Item.Revealing Light] (DC 23) extending outward from a corner of the target's space. You choose which corner of the target's space you want the burst to extend out from at the time you declare the associated Strike. Since the <em>fairy bullet</em> is fired before <em>revealing light</em> can reveal the target, the effects don't affect the flat check for the attack roll with the <em>fairy bullet</em> if the target is @UUID[Compendium.pf2e.conditionitems.Item.Hidden] from you.</p>"
```